test/support/coverage.rb

```diff
@@ -24,7 +24,6 @@ def self.enabled?
     !ENV['COVERAGE'].nil?
   end
 
-  # rubocop:disable Metrics/MethodLength
```

Comment:
The rubocop comments should stay here, as the method can't be shortened. It's not a problem in the code, it's a method dealing with configuration.